main.js

```diff
@@ -30,5 +31,7 @@ client.on('ready', () => {
         }
     });
 });
-
+client.on('message', message => { 
+    EveryoneAndHereMentionMessage.mentionHandler(message);
```

Comment:
Jeśli moduł `EveryoneAndHereMentionMessage` ma tylko jedną eksportowaną funkcję, to po co ją traktować jako metodę? Prościej użyć po prostu `everyoneAndHereMentionMessageHandler` - nazwa (ponownie, z małej litery) zawiera w sobie od razu suffix "Handler". Wtedy [tę funkcję](https://github.com/CodersCommunity/discord-info-bot/pull/4/files#diff-edfa6342d7726b64d1f4ef2715ccb1c7c73438b0d9def94ab1efc3d19db0afe6R14) można wyexportować bezpośrednio z modułu - już bez owijania w obiekt.